src/ToggleButton.svelte

```diff
@@ -0,0 +1,72 @@
+<script lang="ts" context="module">
+  import { capitalize } from "@app/utils";
+
+  export interface ToggleButtonOption<T> {
+    count?: number;
+    title?: string;
+    value: T;
+  }
+</script>
+
+<script lang="ts" strictEvents>
+  type T = $$Generic;
+  import { createEventDispatcher } from "svelte";
+
+  export let options: ToggleButtonOption<T>[];
+  export let active: T;
+
+  const dispatch = createEventDispatcher<{ select: T }>();
+
+  function disabled(count?: number) {
+    return count === undefined || count === 0;
```

Comment:
I think this could be removed and just check against 0 since IMO we wont handle `count === undefined`

Reply:
See https://github.com/radicle-dev/radicle-interface/pull/347#discussion_r958165344.